client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningPersonal.ts

```diff
@@ -7,7 +7,6 @@ import ContactIdValidationSchema from 'Utils/Contacts/ContactIdValidationSchema'
 import { PHONE_NUMBER_REGEX, NOT_REQUIRED_PHONE_NUMBER_REGEX} from 'commons/Regex/Regex';
 
 export const contactQuestioningPersonal = {
-    [InteractedContactFields.IDENTIFICATION_TYPE]: yup.string(),
```

Comment:
why did you remove it?

Reply:
because this validation isn't necessary anymore, there validation in other places in the code